web/src/features/fireWeather/components/WxDataDisplays.tsx

```diff
@@ -96,42 +102,69 @@ export const WxDataDisplays = React.memo(function _(props: WxDataDisplaysProps)
-                  title="Interpolated GDPS noon values: "
-                  rows={noonOnlyGdpsModels}
-                />
-              </ErrorBoundary>
-              <ErrorBoundary>
-                <NoonForecastTable
-                  testId={`noon-forecasts-table-${code}`}
-                  title="Weather forecast noon values: "
-                  rows={noonForecasts}
-                />
-              </ErrorBoundary>
-              <ErrorBoundary>
-                <WxDataGraph
-                  station={station}
-                  timeOfInterest={props.timeOfInterest}
-                  observations={observations}
-                  noonForecasts={noonForecasts}
-                  noonForecastSummaries={noonForecastSummaries}
-                  hrdpsModels={hrdpsModels}
-                  hrdpsSummaries={hrdpsSummaries}
-                  rdpsModels={rdpsModels}
-                  rdpsSummaries={rdpsSummaries}
-                  gdpsModels={gdpsModels}
-                  gdpsSummaries={gdpsSummaries}
-                />
-              </ErrorBoundary>
+              <div>
+                {// Only show tabs after the first station title
+                index === 0 ? (
+                  <Paper square>
+                    <Tabs
+                      className={classes.tabs}
+                      value={value}
+                      onChange={(_changeEvent, tabNumber) => setTabNumber(tabNumber)}
+                      variant="fullWidth"
+                      indicatorColor="primary"
+                      textColor="primary"
+                    >
+                      <Tab label="Tables" onClick={() => toggleTableView(true)} />
+                      <Tab label="Graphs" onClick={() => toggleTableView(false)} />
+                    </Tabs>
+                  </Paper>
+                ) : (
+                  ''
+                )}
+              </div>
+              {showTableView ? (
+                <React.Fragment>
+                  <ErrorBoundary>
+                    <ObservationTable
+                      testId={`observations-table-${code}`}
+                      title="Hourly observations in past 5 days: "
+                      rows={observations}
+                    />
+                  </ErrorBoundary>
+                  <ErrorBoundary>
+                    <NoonModelTable
+                      testId={`noon-gdps-table-${code}`}
+                      title="Interpolated GDPS noon values: "
+                      rows={noonOnlyGdpsModels}
+                    />
+                  </ErrorBoundary>
+                  <ErrorBoundary>
+                    <NoonForecastTable
+                      testId={`noon-forecasts-table-${code}`}
+                      title="Weather forecast noon values: "
+                      rows={noonForecasts}
+                    />
+                  </ErrorBoundary>
+                </React.Fragment>
+              ) : (
+                <React.Fragment>
```

Comment:
You don't have to wrap this with Fragment if there is only one child

Reply:
Oops, copy-pasta, thanks!